### PR TITLE
By user view

### PR DESCRIPTION
**Pull request by mec**
This is the last of the 'by x' views. Here, by user. This one is interesting becuase User is a first  class model so has a slight variation.

The main thing to look out for is the refactor of a mehod name in the authentication concern - this clashes when we start treating 'user' as data and felt more explicit as `current_user`.

The rest follows the pattern, except there is a constraint on the route as it uses a UUID.

https://trello.com/c/hGd6IbH7

image

image

### CODE REVIEW

File: \ [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      []
+    end
+  end
+
+  private def conversion_count_by_user
+    projects = Conversion::Project.not_completed.select(:id, :assigned_to_id)
+    return false unless projects.any?
+
+
+    binding.pry
```
Comment (lozette) [inline \:17]: Uh oh!!!

**Review by lozette — CHANGES_REQUESTED** [review]

File: \ (hunk 1 above)
Comment (lozette) [reply \:17]: How did this pass linting 😆

Comment (mec) [reply \:17]: oops!

Comment (mec) [reply \]: what even is this file!?! deleted!

**Pull request merged**